pydatview/GUISelectionPanel.py

```diff
@@ -448,10 +473,15 @@ def OnAddColumn(self, event):
                 sError=''
                 nError=0
                 for iTab,sTab in zip(ITab,STab):
-                    bValid=main.tabList.get(iTab).addColumnByFormula(sName,sFormula,iFull)
-                    if not bValid:
-                        sError+='The formula didn''t eval for table {}\n'.format(sTab)
-                        nError+=1
+                    if main.selPanel.currentMode == 'sameColumnsMode' or self.parent.tab.active_name == sTab:
```

Comment:
Can you replace the "if" statement with the following:
```python
                haveSameColumns=main.tabList.haveSameColumns(ITab)
                for iTab,sTab in zip(ITab,STab):
                    if haveSameColumns or self.parent.tab.active_name == sTab:
```

Reply:
Sure, it's done. Thanks!